core/src/main/scala/kafka/server/AbstractFetcherManager.scala

```diff
@@ -163,6 +163,16 @@ abstract class AbstractFetcherManager[T <: AbstractFetcherThread](val name: Stri
     info(s"Added fetcher to broker ${fetcherThread.sourceBroker.id} for partitions $initialOffsetAndEpochs")
   }
 
+  def addTopicIdsToFetcherThread(partitionsToUpdate: Set[(TopicPartition, Int)], topicIds: String => Option[Uuid]): Unit = {
```

Comment:
I think I was following the convention of some of the other methods that use `ToFetcherThread` like `addPartitionsToFetcherThread`, but that's a bit different since a parameter is actually a fetcherThread. The update makes sense to me.